pkg/controller.v2/trainjob_controller.go

```diff
@@ -18,14 +18,13 @@ package controllerv2
 
 import (
```

Comment:
Could you group all dependencies like 

- std libs
- external third-party libs
- intenal libs